test/fixtures/run-script/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fastify": "fastify",
```

Comment:
Since this fixture _isn’t_ Fastify, can it get some random name like `cli-tool` or `foo` or something?

Reply:
thanks @anonrig :)

Reply:
Or Ada!!!

Reply:
I’m fine with the name being silly, but seeing as we have a Fastify benchmark somewhere else and some fixtures that really are what their names imply like `typescript.js`, I don’t feel like we should have a “fake” Fastify fixture. We might very well want to add a real Fastify fixture someday.

Reply:
Is everybody ok with me inserting an Ada easter egg?

Reply:
Isn’t Ada the name of the URL library? You probably shouldn’t use a name that actually has meaning within the codebase somewhere, as that can be confusing. Use Ada’s middle name or something.